Makefile

```diff
@@ -65,16 +65,16 @@ set-azure-account:
 	[ "${SKIP_AZURE_LOGIN}" != "true" ] && az account set -s ${AZURE_SUBSCRIPTION} || true
 
 terraform-init: composed-variables set-azure-account
-	$(if $(DOCKER_IMAGE), , $(error Missing environment variable "DOCKER_IMAGE"))
-	$(if $(PULL_REQUEST_NUMBER), $(eval KEY_PREFIX=$(PULL_REQUEST_NUMBER)), $(eval KEY_PREFIX=$(ENVIRONMENT)))
+	$(if $(DOCKER_IMAGE), , $(eval DOCKER_IMAGE="main"))
```

Comment:
I'm not a fan of setting a branch here. lets keep the original where it says it's missing